mp2-v1/tests/common/cases/contract.rs

```diff
@@ -179,19 +173,17 @@ impl ContractController for Vec<MappingUpdate<MappingKey, Address>> {
             match update {
                 MappingUpdate::Deletion(k, _) => {
                     let res = contract.m1(*k).call().await.unwrap();
-                    let v: U256 = res._0.into_word().into();
-                    assert_eq!(v, U256::ZERO, "Key deletion is wrong on contract");
+                    let v: Address = res.0.into();
```

Comment:
Why is `v` an address?

Reply:
Because for some obscure reason this code used to take an `Address`, call [`into_word`] on it turning it into a U256 and compare it with `U256::ZERO`. You can just go ahead and compare the initial `Address` value with `Address::ZERO` then without an expansion to 32 bytes.